lib/driver.rb

```diff
@@ -22,26 +23,55 @@ def initialize(input)
     end
 
     def average_rating
+      return nil if trips.length == 0
+
       total_ratings = 0
       @trips.each do |trip|
         total_ratings += trip.rating
       end
-
-      if trips.length == 0
-        average = 0
-      else
-        average = (total_ratings.to_f) / trips.length
-      end
-
+      average = (total_ratings.to_f) / trips.length
       return average
     end
 
     def add_trip(trip)
       if trip.class != Trip
         raise ArgumentError.new("Can only add trip instance to trip collection")
       end
-
       @trips << trip
     end
+
+    def ignores_incomplete_rides()
+      complete_rides = @trips.select { |trip| trip.end_time != nil }
+      return complete_rides
+    end
```

Comment:
I really like the idea of this helper method - good job identifying it as a responsibility of `Driver`. I'm not quite sure about the name though - something like `completed_trips` might be a little more concise.